TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/Uri/ImageViewHelper.php

```diff
@@ -68,7 +68,7 @@ public function initializeArguments()
     /**
      * Renders the path to a thumbnail image, created from a given image.
      *
-     * @param ImageInterface $image
+     * @param ImageInterface|null $image
```

Comment:
In my test flow did not understand  the ``|null`` properly. If i change this line the change works as expected.